src/openforms/payments/contrib/ogone/plugin.py

```diff
@@ -141,7 +150,8 @@ def apply_status(self, payment, ogone_status) -> None:
         qs = SubmissionPayment.objects.filter(id=payment.id)
         qs = qs.exclude(status__in=PAYMENT_STATUS_FINAL)
         qs = qs.exclude(status=new_status)
-        res = qs.update(status=new_status)
+        update_fields = {"status": new_status, "provider_payment_id": payment_id}
+        res = qs.update(**update_fields)
```

Comment:
```suggestion
        res = qs.update(status=new_status, provider_payment_id=payment_id)
```
the `**kwargs` approach is only required if you're mutating the kwargs before using them, now that the `payment_id` is a required argument, we can remove the indirection

Reply:
whoops, forgot to change it after I made the payment_id mandatory